src/v/hashing/tests/hash_bench.cc

```diff
@@ -11,64 +11,259 @@
 #include "hashing/fnv.h"
 #include "hashing/twang.h"
 #include "hashing/xx.h"
+#include "model/fundamental.h"
+#include "model/ktp.h"
+#include "model/namespace.h"
 #include "random/generators.h"
 
 #include <seastar/core/reactor.hh>
+#include <seastar/core/sstring.hh>
 #include <seastar/testing/perf_tests.hh>
 
+#include <absl/hash/hash.h>
+#include <absl/strings/string_view.h>
 #include <boost/crc.hpp>
 
 static constexpr size_t step_bytes = 57;
 
-PERF_TEST(boost_crc16_fn, header_hash) {
+static constexpr size_t inner_iters = 1000;
+
+template<typename F>
+static size_t header_body(F n) {
```

Comment:
Refactor the identical bodies of the hash benches into this helper to reduce boilerplate.